### ISSUE TITLE
Volume not showing up

### ISSUE THREAD

**Issue by madushan1000** [jrspruitt/ubi_reader#48]
This image(http://72.240.115.5/CGA4131TCH2-P15-20-A000-c2100r172-20180511_PS-D.P7B), has two ubifs volumes(images, rootfs, could be more) embedded in it. But the rootfs volume is slightly unaligned, I guess due to some sort of compression? 
Is it possible to handle cases like this? I was able to extract the volume after adding some padding to align the rootfs volume with the erase count header.

**Comment by jrspruitt** [comment]
Hi madushan1000,

These kind of issues do come up every so often, it seems the manufacturers tweak the driver to do something out of spec. Because there is usually a large amount of times the functions in ubi_reader get called and a variety of things that could be done to get this effect, I've refrained from adding more code for these kind of cases. The code base could get messy and slow very quickly. 

But I have created a different branch, "testing", that is geared more towards debugging and getting through such issues. This is what I use in these cases.

Thanks, 
-Jason
